charts/rclone-copy/templates/rclone-cron.yaml

```diff
@@ -96,6 +99,13 @@ spec:
                     name: {{ $value.name }}
                     key: "{{ $value.key }}"
             {{- end }}
+            {{- range $key, $value := .Values.passwordSecrets }}
+              - name: {{ printf "UNOBSCURED_%s" $key | quote }}
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ $value.name }}
+                    key: "{{ $value.key }}"
+            {{- end }}
```

Comment:
I'm probably missing something but couldn't we obscure the secrets here right away? I find the `printf "export %s=` above a little clunky.

Reply:
Nevermind, we're not do not and don't want to dereference the secret value here.

Reply:
This is just configure env variables. We cannot execute commands here